src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -1131,17 +1145,35 @@ private static byte[] getData(Profile p, int tagSignature) {
      * This method is useful for advanced applications which need to access
      * profile data directly.
      *
+     * <p>
+     * Note: JDK built-in ICC Profiles cannot be updated using this method
+     * as it will result in {@code IllegalArgumentException}. JDK built-in
+     * profiles are those obtained by {@code ICC_Profile.getInstance(int colorSpaceID)}
+     * where {@code colorSpaceID} is one of the following:
+     * {@link ColorSpace#CS_sRGB}, {@link ColorSpace#CS_LINEAR_RGB},
+     * {@link ColorSpace#CS_PYCC}, {@link ColorSpace#CS_GRAY} or
+     * {@link ColorSpace#CS_CIEXYZ}.
+     *
      * @param  tagSignature the ICC tag signature for the data element you want
      *         to set
      * @param  tagData the data to set for the specified tag signature
      * @throws IllegalArgumentException if {@code tagSignature} is not a
      *         signature as defined in the ICC specification.
-     * @throws IllegalArgumentException if a content of the {@code tagData}
+     * @throws IllegalArgumentException if the content of the {@code tagData}
      *         array can not be interpreted as valid tag data, corresponding to
      *         the {@code tagSignature}
+     * @throws IllegalArgumentException if this is a built-in profile for one
+     *         of the pre-defined color spaces, that is those which can be obtained
+     *         by calling {@code ICC_Profile.getInstance(int colorSpaceID)}
      * @see #getData
+     * @see ColorSpace
      */
     public void setData(int tagSignature, byte[] tagData) {
+        if (builtIn) {
+            throw new IllegalArgumentException("Built-in profile"
+                                               + " cannot be modified");
```

Comment:
Is it acceptable to overshoot 80-column limit for 6 characters without wrapping the line?